common/views/components/NewsletterSignup/NewsletterSignup.js

```diff
@@ -20,33 +20,29 @@ type State = {|
 const addressBooks = [
   {
     id: 'whats_on',
-    label: `What’s On`,
-    name: 'addressbook_40131',
-    description: `Exhibitions, events and opportunities to get involved - What’s On is sent monthly, with occasional updates`
+    label: `<span class="${font({s: 'HNL5'})}"><span class="${font({s: 'HNM5'})}">What’s On</span> at Wellcome Collection: our monthly roundup of the latest exhibitions, events, new books and opportunities to get involved. Sent monthly with up to one extra update per month.</span>`,
+    name: 'addressbook_40131'
   },
   {
     id: 'accessibility',
-    label: `What’s On: Access`,
-    name: 'addressbook_40129',
-    description: `What’s On: Access - highlights events, tours and opportunities to get involved, including BSL, Audio Description and Speech-to-Text events`
+    label: `<span class="${font({s: 'HNL5'})}"><span class="${font({s: 'HNM5'})}">Access</span> events, tours and opportunities to get involved, including British Sign Language, Audio Description and Speech-To-Text activities. Sent quarterly with occasional updates.</span>`,
+    name: 'addressbook_40129'
   },
   {
     id: 'young_people_14-19',
-    label: `What’s On for 14-19 year olds`,
-    name: 'addressbook_40132',
-    description: `What’s On for 14-19 year olds: creative opportunities and events for young people aged 14-19, including RawMinds and Saturday Studio`
+    label: `<span class="${font({s: 'HNL5'})}">Events and opportunities to get involved for <span class="${font({s: 'HNM5'})}">14-to-19-year-olds</span>, including RawMinds and Saturday Studios. Sent monthly with occasional updates.</span>`,
```

Comment:
Why can't we set the font on `<b>` to `HNM5`?

Reply:
The browser bolding is inconsistent (and looks crummy at the best of times):

![screen shot 2018-05-17 at 11 43 51](https://user-images.githubusercontent.com/1394592/40172788-9bbf9888-59c7-11e8-9add-ec1deb9a6a4d.png)